Include/cpython/object.h

```diff
@@ -256,6 +256,9 @@ typedef struct _typeobject {
     destructor tp_finalize;
     vectorcallfunc tp_vectorcall;
 
+    /* Unused, kept for backwards compatibility in CPython 3.8 only */
```

Comment:
Please mention that it will be removed from Python 3.8. You may add "bpo-37250: " prefix.